packages/graph-explorer/src/hooks/useDeepMemo.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable react-compiler/react-compiler */
```

Comment:
Whatever was causing the ESLint error before has been resolved in the latest version of the ESLint plugin.